src/components/ItemData/ItemData.js

```diff
@@ -0,0 +1,35 @@
+import Burger from '../../util/images/Burger.jpg';
```

Comment:
Mock data should be place in mocks directory so our components serve the purpose of it's name ..